crates/cdk-mintd/src/main.rs

```diff
@@ -148,6 +148,7 @@ async fn main() -> anyhow::Result<()> {
                 CurrencyUnit::Sat,
                 PaymentMethod::Bolt11,
                 mint_melt_limits,
+                Some(false),
```

Comment:
CLN and some of the other backends can support paying amountless invoices, so that should be updated to actually support this change where we can and not just support the new field but disable it.